.flaky8

```diff
@@ -0,0 +1,12 @@
+[flake8]
+ignore =
+  # Closing bracket mismatches opening bracket's line.
+  # This works poorly with type annotations in method declarations.
+  E123, E124
+  # Line over-indented for visual indent.
+  # This works poorly with type annotations in method declarations.
+  E126, E128, E131
+  # Line break after binary operator.
+  # This catches line breaks after "and" / "or" as a means of breaking up
+  # long if statements, which PEP 8 explicitly encourages.
+  W504
```

Comment:
I'd like to add my vote for just using [black](https://github.com/psf/black) to enforce styling and formatting, but that is outside the scope of this review.

Reply:
Thanks! Will bring the discussion to #3 and update from there.